docs/configuration/OVERVIEW.md

```diff
@@ -86,50 +87,85 @@ components at their instantiation time
 
 ### Dynamic configuration
 
-- The default configuration is extracted by __cms-adapters__ tool and bundled in a JSON file will be exposed in the CMS
+- The default configuration is extracted by __@o3r/components:extractor__ and bundled in a JSON file will be exposed in the CMS
 
-- In the CMS plugin the Business Analyst will modify configuration of components and this ones will be exposed (in the
+- In the CMS plugin the Business Analyst will modify the configuration of components which will be exposed (in the
   server) as dynamic content of the app
 
-- The application will do an HTTP call, get the custom config and inject it in the store.
+- The application will do an HTTP call to get the custom config and inject it in the store.
 
-__No need of rebuild/redeploy the app__
+__No need to rebuild/redeploy the app__
 
 ## Config types lib/app
 
 - At __library__ level we have the config for components (__blocks, components, elements__).
```

Comment:
Make a reference to the page where these terms are defined

Reply:
It is a bit weird you only mention an example with a block component, but this is not the only case. The pattern is also true for exposed components etc.